packages/google_adsense/lib/src/ad_unit_widget_web.dart

```diff
@@ -0,0 +1,201 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:developer';
+import 'dart:js_interop';
+import 'dart:js_interop_unsafe';
+
+import 'package:flutter/widgets.dart';
+import 'package:web/web.dart' as web;
+
+import '../google_adsense.dart';
+import 'js_interop/adsbygoogle.dart';
+
+/// Widget displaying an ad unit
+class AdUnitWidgetWeb extends AdUnitWidget {
+  // TODO(sokoloff06): consider builder?
+  /// Constructs [AdUnitWidgetWeb]
+  AdUnitWidgetWeb({
+    required String adClient,
+    required String adSlot,
+    required bool isAdTest,
+    required Map<String, dynamic> additionalParams,
+    String? cssText,
```

Comment:
What do you think about making these values a configuration object instead? So we end up with:

```dart
AdUnitWidget(
  AdUnitConfig(
    adClient: "my_client_id",
    adSlot: "slot_name",
    additionalParams: AdUnitAdditionalParams(
      fullWidthResponsive: true,
      adFormat: AdFormat.SOME_TYPE,
    ),
  )
);
```

Reply:
Created AdUnitConfiguration class and updated the API